Example/Example/ViewControllers/DemoViewController.swift

```diff
@@ -0,0 +1,134 @@
+import UIKit
+import SalemoveSDK
+
+class DemoViewController: UIViewController {
+    @IBOutlet weak var engagementButton: UIButton!
+    @IBOutlet weak var configurationButton: UIButton!
+
+    private var statusViewController: EngagementStatusViewController?
+
+    // MARK: System
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        Salemove.sharedInstance.pushHandler = {
+            push in
+
```

Comment:
Can you show an example on how they can handle this notification?

Reply:
Can be done in next iteration

Reply:
When is the next iteration?